### DISCUSSION TITLE
expected behavior? -- tab content focus state

### DISCUSSION THREAD

**Discussion by elainelylee** [twilio-labs/paste#995] [Q&A] [answered]
Is the content expected to have a focus state around it? 

![image](https://user-images.githubusercontent.com/6718764/101061254-27b32700-3545-11eb-9ac3-ec06f51a6194.png)

**Comment by TheSisb** [comment, marked as answer]
Hi @elainelylee,

Yes this is expected behavior as per https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html
Quote:
```
- Puts the tabpanel in the page Tab sequence.
- Facilitates movement to panel content for assistive technology users.
- Especially helpful if there are panels that do not contain a focusable element.
```

**Comment by elainelylee** [comment]
Trying to understand, should this also apply to a component like the
Disclosure?

Elaine Lee
Principal Product Designer - Twilio AI
[image: twilio] <https://www.twilio.com/?utm_source=email_signature>
EMAIL ellee@twilio.com

**Reply by TheSisb** [reply]
Good question! Not seeing that as a requirement in the spec so I'm guessing no. cc @SiTaggart if you have more insight as to why not?

**Reply by SiTaggart** [reply]
Tab panel being focusable is certainly a new addition to the spec / guidance. 

The fact that disclosures don't follow the same advice doesn't necessarily surprise me. This is a bit of an educated guess as I don't know the exact reason they added it, but the reason it doesn't surprise me is the difference in the way disclosures and tab sets handle the focusability (is that word?) of their triggers.

If you have a tab set with 6 tabs, the entire component is only 2 tab stops. The active tab and the tab panel.

If you have a collection of 6 disclosures, you'd have 12 tab stops if the disclosure content was also focusable. It would be kind of annoying.

That guidance might also change, so we'll keep an eye on it. What we do now seems to be sensible to me, and follows the specs as closely as we can without too many inconveniences to keyboard users. Hope that helps clarify a little more.
